### ISSUE TITLE
SEO: Configure `robots.txt` file.

### ISSUE THREAD

**Issue by jimtyhurst** [Marmalade-ai/company-website#130]
## Expected Behavior
We want the company website to score as high as possible in the Lighthouse SEO scoring.

## Actual Behavior
One of the Gatsby Cloud [build reports](https://build-1af2efe3-37c6-4b51-b75f-495fbac4d79f.gtsb.io/reports/lighthouse/index.html) from Lighthouse dinged our code a few points on SEO, saying:
```
Search engines are unable to include your pages in search results if they don't have permission to crawl them.
...
Blocking Directive Source
x-robots-tag: none
```
## Analysis
* I looked at the `meta` tags in the build. I do not see that `x-robots-tag` anywhere. Maybe it is `none` by default? Maybe we need to explicitly set that tag.
* See [gatsby-plugin-robots-txt](https://www.gatsbyjs.com/plugins/gatsby-plugin-robots-txt/) as a way to create `robots.txt` from metadata in [gatsby-config.js](./gatsby-config.js)

**Comment by craigfisk** [comment]
If I understand correctly (from the first link below), Lighthouse runs on
the results of indexing, and you wouldn't want the Gatsby cloud version to
be indexed, so what you are seeing is ok, and Lighthouse will work with the
https://marmalade.ai version only. Is that the way you see it, too?

1)
https://answers.netlify.com/t/disabling-x-robots-tag-header-on-hosted-site/8857
2)
https://www.google.com/search?q=%22x-robots-tag%22+gatsby+lighthouse&oq=%22x-robots-tag%22+gatsby+lighthouse&aqs=chrome..69i57.34455j0j7&client=ms-android-google&sourceid=chrome-mobile&ie=UTF-8#scso=_jdJFYeq7C8G2tAa3y7boBQ14:0

https://www.linkedin.com/in/craigfisk <https://about.me/craigfisk>
(650) 336-8770

**Comment by jimtyhurst** [comment]
Yes, I was assuming that whatever was in the branch build would be deployed, but it makes sense that it would be different in this case. I will leave this issue open until we deploy the new company website to production, so we can verify that the new site is indexed correctly.

See also: [gatsby-plugin-next-seo](https://www.gatsbyjs.com/plugins/gatsby-plugin-next-seo/?=seo#no-index).
